src/analysis/locally_convex/basic.lean

```diff
@@ -151,6 +158,18 @@ begin
   exact ⟨hA _ ha ⟨_, hx₁, rfl⟩, hB _ ha ⟨_, hx₂, rfl⟩⟩,
 end
 
+lemma balanced.Inter_finset {t : finset ι} {f : ι → set E}
```

Comment:
```suggestion
lemma balanced_Inter_finset {t : finset ι} {f : ι → set E}
```
because there's no dot notation possible on it. See `convex_Inter`. Also, is this not true for arbitrary intersections? and arbitrary unions as well?